pom.xml

```diff
@@ -38,10 +38,6 @@
     <node.version>22.15.0</node.version>
     <npm.version>11.3.0</npm.version>
     <spotless.check.skip>false</spotless.check.skip>
-
-    <!-- Opt in to Jenkins handling eslint results -->
-    <maven.test.failure.ignore>false</maven.test.failure.ignore>
-    <frontend.testFailureIgnore>${maven.test.failure.ignore}</frontend.testFailureIgnore>
```

Comment:
We need to restore this as well

Reply:
This is part of the magic.

If I restore this, then the Jenkins pipeline will no longer fail for Prettier and TypeScript issues.

That's because the Jenkins pipeline seems to be called with `-Dmaven.test.failure.ignore=true`, which gets propagated to `frontend.testFailureIgnore` through these lines.

By not allowing Jenkins to tweak `frontend.testFailureIgnore`, I can control which specific command should have its exit code ignored (eslint and vitest) through the `npm` script.